src/consumer/kafka.rs

```diff
@@ -71,6 +82,56 @@ pub async fn start_consumer(
     .await
 }
 
+async fn create_missing_topics(
```

Comment:
Let's move this to another file since I don't think it belongs in the consumer directory.